docs/integrations/prefect-kubernetes/index.mdx

```diff
@@ -104,32 +117,6 @@ def kubernetes_orchestrator():
     )
 ```
 
-### Patch an existing Kubernetes deployment
```

Comment:
because its using that old removed utility and I wanted to get the fix in

I'm happy to reintroduce a valid "patch an existing job" example in a following PR